resources/assets/uad_lists.json

```diff
@@ -10520,15 +10520,6 @@
     "labels": null,
     "removal": "Expert"
   },
-  {
```

Comment:
Can you tell why you removed this? Is it no longer existing?

Reply:
I changed the listing for this app, since it can be removed safely

Reply:
Alright, thanks!